### DISCUSSION TITLE
how to run test X times?

### DISCUSSION THREAD

**Discussion by kildarr** [warmuuh/milkman#125] [Q&A] [answered]
How do I execute the same test say 100 times?  I don't see any options for this?  Isn't that part of test runner?  And no I don't want to drag it to tests 100 times.

**Comment by warmuuh** [comment, marked as answer]
Maybe drag in 10 and click 10 times :D

To be serious: no, there is no such feature yet, let me think about adding it

**Reply by warmuuh** [reply]
implemented on nightly. see #126
